app/routes.py

```diff
@@ -32,39 +32,40 @@ def checker_page():
     return make_response(jsonify({'success': True, 'message': 'OK', 'version': APP_VERSION}))
 
 
-@app.route('', methods=['GET'])
-@app.route('/', methods=['GET'])
+@app.route('/iconsets', methods=['GET'])
 def all_icon_sets():
     return make_response(jsonify({"success": True, "items": get_all_icon_sets()}))
 
 
-@app.route('/<string:icon_set_name>', methods=['GET'])
+@app.route('/iconset/<string:icon_set_name>', methods=['GET'])
 def icon_set_metadata(icon_set_name):
     icon_set = get_and_check_icon_set(icon_set_name)
     return make_response(jsonify(icon_set))
```

Comment:
Looking at the API guideline: https://github.com/geoadmin/doc-guidelines/blob/master/API.md
Here we should have something like this:
```
{"success": True, **jsonify(icon_set)}
```